src/scss/_ansi.scss

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+/* ansi text */
+.ansi-bold {
+  font-weight: bold;
+}
+
+.ansi-faint {
+  // cannot use sass variables in functions
+  color: darken(hsl(0, 0%, 98%), 50%);
+}
+
```

Comment:
I couldn't use a func substitution variable here... :/

Reply:
i would propose that we should probably still define these in `_variables` so we have all the colors defined in one place.